utf8.h

```diff
@@ -1341,13 +1341,13 @@ typedef enum {
  * there is a NN assertion for it.  It causes that to pass but to still
  * segfault if wrongly gets used */
 #define Perl_utf8_to_bytes_overwrite(mTHX, s, l)                            \
-        Perl_utf8_to_bytes_(aTHX_ s, l, INT2PTR(U8 **, 1),                  \
+        Perl_utf8_to_bytes_(aTHX_ s, l, INT2PTR(void **, 1),                \
                                   PL_utf8_to_bytes_overwrite)
 #define Perl_utf8_to_bytes_new_pv(mTHX, s, l, f)                            \
         Perl_utf8_to_bytes_(aTHX_ (U8 **) s, l, f,                          \
                                   PL_utf8_to_bytes_new_memory)
 #define Perl_utf8_to_bytes_temp_pv(mTHX, s, l)                              \
-        Perl_utf8_to_bytes_(aTHX_ (U8 **) s, l, INT2PTR(U8 **, 1),          \
+        Perl_utf8_to_bytes_(aTHX_ (U8 **) s, l, INT2PTR(void **, 1),        \
```

Comment:
Not a new issue, but a caller can pass a `STRLEN **` as `s` and this will cast it away.

The `1` pointer is still bad.

Reply:
Also `s` should have parens around it, since it might not be a simple `&ptr`:
```
U8 **my_strings = ...;
for (i = 0; i < my_string_count; ++i) {
  utf8_to_bytes_temp_pv(i+my_strings, i+my_lens); // oops
}
```